### PR TITLE
load multi level subfolders

### PR DESCRIPTION
**Pull request by zinks-**
Now subfolders of subfolders can be loaded
Sent
 > Foo
 > Foo/Bar
 > Foo/Bar/barfoo

The window title says "undefined", but still better than not loading the messages!

### CODE REVIEW

**Comment by zinks-** [comment]
@jancborchardt @DeepDiver1975 @Xenopathic 
Should be able to load any depth of subfolders with all directory separators, please test

File: js/views/folder.js [hunk 1]
```diff
@@ -83,10 +82,15 @@ views.Folders = Backbone.Marionette.CollectionView.extend({
 		var activeFolder = activeAccount;
```
Comment (Xenopathic) [inline js/views/folder.js]: You'll probably want to use the mail delimiter on the line above this one too (`folderId.split()`)

**Comment by zinks-** [comment]
@Xenopathic thanks for noticing, fixed

**Comment by zinks-** [comment]
closing this for now as there are so many things wrong with what I'm doing here !

**Comment by Xenopathic** [comment]
@zinks- Are there? It seemed quite sensible to me

**Comment by clementhk** [comment]
@zinks- there are so many things wrong in the original proposal as well :) I guess we'll just have to live with it for now.

**Pull request closed without merging**